src/clusterfuzz/_internal/platforms/android/fetch_artifact.py

```diff
@@ -100,16 +100,24 @@ def get_artifacts_for_build(client, bid, target, attempt_id='latest'):
   request = client.buildartifact().list(
       buildId=bid, target=target, attemptId=attempt_id)
 
+  request_str = '{}, {}, {}, {}'.format(request.uri, request.method,
```

Comment:
Can you define this closer to where it's used? i.e. right before line 118. 

We can also simplify this by doing a `', '.join([request.uri, ...])` instead.

Reply:
I tried testing it in a python shell, and it seems like to be able to use `.join`, all the elements of the list should be strings.
`", ".join(['abc', [1, 2, 3], {'a':1, 'b':2}])` fails, but `"{}, {}, {}".format('abc', [1, 2, 3], {'a':1, 'b':2})` works.
I am not certain if all the items would be strings in this case.

Reply:
I had to do it before the `request` was used up. If I understand the existing code correctly, it is being used as an iterable. Therefore, if I used it closer to the logging line, that we may not get the full `request` object.

Reply:
Use an f-string instead of format.